README.md

```diff
@@ -1 +1,47 @@
-# openapi-ts-template
\ No newline at end of file
+# openapi-ts-template
+
+## Description
+
+A `Node.js` typescript template with open api implementation 
+
+## Configuration
+
+Use a `.env` at root of the repository to set values for the environment variables defined in [custom-environment-variables.json](./config/custom-environment-variables.json).
```

Comment:
few mentions of `/config` in readme but that's gone now

Reply:
Readme has been updated.